### PR TITLE
Ships now speed up and modernized README

### PR DESCRIPTION
**Pull request by driechers**

### CODE REVIEW

**Comment by stokito** [comment]
Hi, thank for you for contribution :)
Unfortunately I'm not an official maintainer, I just cloned the repo from Bazaar repo to have a copy which might be easily searched.
Also please note that your patch is incorrect for now, since all changes should be made via patches in [debian/patches](https://github.com/stokito/grub-invaders/tree/master/debian/patches) directory.
Idea is to keep original source code as it was in on Eric's tarball  http://www.erikyyy.de/invaders/
It's probably because there weren't any standard VCS in that times.

But personally I think that it's not necessary for now and source  should be in actual state

**Comment by driechers** [comment]
Ah! Thanks for letting me know.

On Sat, Jan 21, 2017 at 10:25 PM, Sergey Ponomarev <notifications@github.com

**Comment by stokito** [comment]
I created a #4 ticket

File: game.c [hunk 1]
```diff
@@ -252,7 +255,7 @@ void sounder()
       }else{
 	sound_nosound();
       };
-      delay_wait(1193180/80);
+      delay_wait(1193180/1000+livealiens*500);
```
Comment (fabiangreffrath) [inline game.c:258]: This formula somehow looks wrong to me. The original value is `14914`. With this new formula the start value is now `21193` which makes the game quite a bit slower and then it decreases to an insane `1693` when there is only one ship left.

I think game speed should start with the original value and then converge to something like 2x speed when there is only 1 ship left. Something like `1193180/(160-2*livealiens)` probably? What do you think?

**Review by fabiangreffrath — COMMENTED** [review]
Thoughts?

**Comment by stokito** [comment]
Hi Fabian, since the author didn't answered to me then you can feel free to create your own fork, apply all those patches from debian folder and write to Debian maintainers list that you would like to become a new maintainer and release the package.

**Comment by fabiangreffrath** [comment]
I am the Debian maintainer. :wink:

**Pull request merged**